Websites/perf.webkit.org/public/v3/components/pane-selector.js

```diff
@@ -49,6 +49,8 @@ class PaneSelector extends ComponentBase {
 
             if (currentMetric) {
                 for (var platform of Platform.sortByName(Platform.all())) {
+                    if (platform.isHidden())
```

Comment:
We probably want to do the same filtering in ChartsPage's alternatePlatforms function.

Reply:
Nice catch.